### PR TITLE
Expand `debugError` stub

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Refs #640.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/common/msg_debugerror.go [hunk 1]
```diff
@@ -18,11 +18,27 @@ import (
...
+	if err != nil {
+		return nil, err
+	}
+
+	switch expected {
+	case "panic":
+		panic("oops!")
+	default:
+		return nil, errors.New(expected)
+	}
```
Comment (chilagrow) [inline internal/handlers/common/msg_debugerror.go:43]: We'd want a test each cases.

**Review by chilagrow — CHANGES_REQUESTED** [review]

File: internal/handlers/common/msg_debugerror.go (hunk 1 above)
Comment (AlekSi) [reply internal/handlers/common/msg_debugerror.go:43]: Sure, that's for someone who would actually do that task :) That's just a start to make it easier to estimate

Comment (chilagrow) [reply internal/handlers/common/msg_debugerror.go:43]: Ohh, it almost looked like this task was done :see_no_evil:

**Comment by AlekSi** [comment]
@chilagrow PTAL

**Review by chilagrow — APPROVED** [review]
:+1: the part that requires clear

File: internal/handlers/common/msg_debugerror.go [hunk 2]
```diff
@@ -18,11 +18,27 @@ import (
...
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	expected, err := GetRequiredParam[string](document, document.Command())
+	if err != nil {
+		return nil, err
+	}
+
 	// TODO https://github.com/FerretDB/FerretDB/issues/640
```
Comment (rumyantseva) [inline internal/handlers/common/msg_debugerror.go:37]: Why is TODO still actual?

File: internal/handlers/common/msg_debugerror.go [hunk 3]
```diff
@@ -18,11 +18,27 @@ import (
...
-	return nil, errors.New("debugError")
+	switch expected {
+	case "panic":
+		panic("oops!")
```
Comment (rumyantseva) [inline internal/handlers/common/msg_debugerror.go:40]: Do we want a more meaningful message here?

**Review by rumyantseva — APPROVED** [review]
Looks like a draft to me. If the purpose of it was to have a draft, LGTM. Otherwise, doesn't seem to be ready (doesn't do what the task describes etc)

File: internal/handlers/common/msg_debugerror.go (hunk 2 above)
Comment (AlekSi) [reply internal/handlers/common/msg_debugerror.go:37]: Yes. That PR does not close this issue.

File: internal/handlers/common/msg_debugerror.go (hunk 3 above)
Comment (AlekSi) [reply internal/handlers/common/msg_debugerror.go:40]: Maybe. It could be changed during implementation of the issue.

**Comment by AlekSi** [comment]
It does not close the issue. It wasn't intended for that issue to do that. That's also not a draft.

**Pull request merged**